dashboard/src/components/OperatorNew/OperatorNew.v2.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useEffect, useState } from "react";
+
+import actions from "actions";
+import { CdsButton } from "components/Clarity/clarity";
+import Alert from "components/js/Alert";
+import Column from "components/js/Column";
+import Row from "components/js/Row";
+import OperatorNotSupported from "components/OperatorList/OperatorsNotSupported.v2";
+import OperatorSummary from "components/OperatorSummary/OperatorSummary";
+import { push, RouterAction } from "connected-react-router";
+import { useDispatch, useSelector } from "react-redux";
+import { Action } from "redux";
+import { ThunkDispatch } from "redux-thunk";
+import { IOperatorsStateError } from "../../reducers/operators";
+import { Operators } from "../../shared/Operators";
+import { IPackageManifest, IPackageManifestChannel, IStoreState } from "../../shared/types";
+import { api, app } from "../../shared/url";
+import LoadingWrapper from "../LoadingWrapper/LoadingWrapper.v2";
+import OperatorHeader from "../OperatorView/OperatorHeader.v2";
+import "./OperatorNew.v2.css";
+
+interface IOperatorNewProps {
+  operatorName: string;
+  operator?: IPackageManifest;
+  getOperator: (namespace: string, name: string) => Promise<void>;
+  isFetching: boolean;
+  cluster: string;
+  namespace: string;
+  errors: IOperatorsStateError;
+  createOperator: (
+    namespace: string,
+    name: string,
+    channel: string,
+    installPlanApproval: string,
+    csv: string,
+  ) => Promise<boolean>;
+  push: (location: string) => RouterAction;
+}
+
+export default function OperatorNew({ namespace, operatorName, cluster }: IOperatorNewProps) {
+  const dispatch: ThunkDispatch<IStoreState, null, Action> = useDispatch();
+
+  const [updateChannel, setUpdateChannel] = useState(
+    undefined as IPackageManifestChannel | undefined,
+  );
+  const [updateChannelGlobal, setUpdateChannelGlobal] = useState(false);
+  // Instalation mode: true for global, false for namespaced
+  const [installationModeGlobal, setInstallationModeGlobal] = useState(false);
+  // Approval strategy: true for automatic, false for manual
+  const [approvalStrategyAutomatic, setApprovalStrategyAutomatic] = useState(true);
+
+  useEffect(() => {
+    dispatch(actions.operators.getOperator(namespace, operatorName));
+  }, [dispatch, namespace, operatorName]);
+
+  const {
+    operator,
+    isFetching,
+    errors: { operator: errors },
+  } = useSelector((state: IStoreState) => state.operators);
+
+  useEffect(() => {
+    if (operator) {
+      const defaultChannel = Operators.getDefaultChannel(operator);
+      const global = Operators.global(defaultChannel);
+      setUpdateChannel(defaultChannel);
+      setUpdateChannelGlobal(global);
+      setInstallationModeGlobal(global);
+    }
+  }, [operator]);
+
+  if (cluster !== "default") {
```

Comment:
I think you've already noted that it'd be good to update these to use the `config.kubeappsCluster` instead.